src/components/Footer.jsx

```diff
@@ -12,8 +12,8 @@ const Footer = () => (
     <div id="footer-text" className="col-xs-6 col-sm-10">
       <p>
         <small>
-          Sinopia is a project of <a rel="grant noopener noreferrer" href="http://www.ld4p.org" target="_blank">
-          Linked Data for Production 2 (LD4P2)</a>, generously funded by the Andrew W. Mellon Foundation.
+          Sinopia is a project of <a rel="grant noopener noreferrer" href="http://www.ld4p.org" target="_blank">Linked Data for Production 2 (LD4P2)</a>,
+          generously funded by the Andrew W. Mellon Foundation.
```

Comment:
i liked the old version better for this one, but i agree with the upgraded linter on the rest of them